daemon/mgr/image_utils.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/alibaba/pouch/pkg/reference"
 )
 
-// addRegistry add default registry if needed.
+// addRegistry add default registry and namespace if needed.
```

Comment:
Could you help to add  unit test cases for this function? @Ace-Tang

Reply:
Sure.